gdx/src/com/badlogic/gdx/graphics/g3d/particles/ParticleChannels.java

```diff
@@ -83,6 +83,20 @@ public void init (FloatChannel channel) {
 		}
 	}
 
+	public static class Scale3Initializer implements ChannelInitializer<FloatChannel> {
```

Comment:
Correct! Sorry, I missed it during one of the renaming sessions I had.
I'll run a sweep through the other classes to check if I missed anything else!